docs/resources/Guild.md

```diff
@@ -571,6 +571,20 @@ Returns a list of [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) obje
 | limit | integer   | max number of members to return (1-1000) | 1       |
 | after | snowflake | the highest user id in the previous page | 0       |
 
+## Search Guild Members % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/members/search
+Returns a list of [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) objects whose username or nickname starts with a provided string.
+
+> info
+> All parameters to this endpoint except for `query` are optional
+
+###### Query String Params
+
+| Field | Type    | Description                                                | Default |
+| ----- | ------- | ---------------------------------------------------------- | ------- |
+| query | string  | Query string to match username(s) and nickname(s) against. |         |
+| limit | integer | max number of members to return (1-1000)                   | 1       |
```

Comment:
The limit is 100, not 1000.

Reply:
Are you sure the limit's 100? I seem to be getting 1000 back from this endpoint for the upper bounds